cocotb_build_libs.py

```diff
@@ -19,6 +19,56 @@
 cocotb_share_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "cocotb", "share"))
 
 
+def create_rc_file(name, filename, libraries):
```

Comment:
I think this needs a comment linking to the docs on what exactly an rc file is.

Reply:
I have added comments and links to the documentation provided by Microsoft.